ONSdigital/ssdc-rm-support-tool
# 983 reschedule action Rules

# Motivation and Context
An easier way to reschedule action rules would make our lives easier

# What has changed
UI:
- Added an extra button in support tools when looking at actions
- Button will open a dialog where you can input the new date and time
- A confirm box will open to double check you wish to change the time

Endpoint:
 - Added a 'put' method in the support tool back end point to update trigger time

Added a test case


# How to test?
Run the code locally in docker dev and see if it updates the trigger time
Check the UI is intuitive 

# Links
https://trello.com/c/Y9s01znI

# Screenshots (if appropriate):
